src/test/java/redis/clients/jedis/modules/RedisModulesPipelineTest.java

```diff
@@ -0,0 +1,82 @@
+package redis.clients.jedis.modules;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import redis.clients.jedis.DefaultJedisClientConfig;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.Pipeline;
+import redis.clients.jedis.Response;
+import redis.clients.jedis.json.Path;
+import redis.clients.jedis.search.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static redis.clients.jedis.search.RediSearchUtil.toStringMap;
+
+public class RedisModulesPipelineTest extends RedisModuleCommandsTestBase {
+    @BeforeClass
+    public static void prepare() {
+        RedisModuleCommandsTestBase.prepare();
+    }
+
+    @Test
+    public void search() {
+        Jedis jedis = new Jedis(hnp, DefaultJedisClientConfig.builder().timeoutMillis(500).build());
+        jedis.flushAll();
+
```

Comment:
```suggestion
```

https://github.com/redis/jedis/blob/d45e0cf318bc1f95703636ed3d32b2245d298642/src/test/java/redis/clients/jedis/modules/RedisModuleCommandsTestBase.java#L36-L42

should cover it.

Reply:
It's not :sweat_smile: , client (UnifiedJedis) does not have Pipeline